test/integration/src/test/java/io/pravega/test/integration/controller/server/StreamMetadataTest.java

```diff
@@ -83,6 +103,35 @@ public void testMetadataOperations() throws Exception {
                                                                .scalingPolicy(scalingPolicy)
                                                                .build();
 
+        // Create stream and check rollover size
+        // Create segment helper
+        HostMonitorConfig hostMonitorConfig = HostMonitorConfigImpl.builder()
+                .hostMonitorEnabled(false)
+                .hostMonitorMinRebalanceInterval(10)
+                .containerCount(4)
+                .hostContainerMap(HostMonitorConfigImpl.getHostContainerMap(ENDPOINT, servicePort, 4))
+                .build();
+        HostControllerStore hostStore = HostStoreFactory.createInMemoryStore(hostMonitorConfig);
```

Comment:
None of these objects are cleaned up. You need to close them (use `@Cleanup`) to release any resources they acquired during their lifetime.